test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,30 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // empty list
+        assertNoNull();
+
+        // some null objects
+        assertHasNull((Object) null);
+        assertHasNull(1, null);
+        assertHasNull("a", null, "c");
+        assertHasNull(1, null, 2, null, 3);
```

Comment:
Quite thorough, however there are some redundancy.
Does each additional test case have chance of finding a new bug?